plugin.video.otaku/resources/lib/AniListBrowser.py

```diff
@@ -3248,6 +3265,80 @@ def get_relations_res(self, variables):
         json_res = results['data']['Media']['relations']
         return json_res
 
+    def get_watch_order_res(self, variables):
+        query = '''
+            query ($idMal: Int) {
+                Media (idMal: $idMal, type: ANIME) {
+                    id
+                    idMal
+                    title {
+                        userPreferred,
+                        romaji,
+                        english
+                    }
+                    coverImage {
+                        extraLarge
+                    }
+                    bannerImage
+                    startDate {
+                        year,
+                        month,
+                        day
+                    }
+                    description
+                    synonyms
+                    format
+                    episodes
+                    status
+                    genres
+                    duration
+                    countryOfOrigin
+                    averageScore
+                    characters (
+                        page: 1,
+                        sort: ROLE,
+                        perPage: 10,
+                    ) {
+                        edges {
+                            node {
+                                name {
+                                    userPreferred
+                                }
+                            }
+                            voiceActors (language: JAPANESE) {
+                                name {
+                                    userPreferred
+                                }
+                                image {
+                                    large
+                                }
+                            }
+                        }
+                    }
+                    studios {
+                        edges {
+                            node {
+                                name
+                            }
+                        }
+                    }
+                    trailer {
+                        id
+                        site
+                    }
+                }
+            }
+            '''
+        result = client.request(self._URL, post={'query': query, 'variables': variables}, jpost=True)
+        if result != None:
```

Comment:
Chiaki doesn't differentiate between ANIME and other media. Hence the != None check since we've got type: ANIME in our filter.